frontend/src/store/modules/LookupStore.ts

```diff
@@ -150,6 +154,9 @@ export default {
     sobonOrientierungswertJahr(state: LookupState, list: LookupEntryDto[]): void {
       state.sobonOrientierungswertJahr = list;
     },
+    sobonOrientierungswertJahrWithoutStandortabfrage(state: LookupState, list: LookupEntryDto[]): void {
```

Comment:
Da die meisten Variablen auch eher eine Zusammensetzung deutscher Wörter sind, würde ich das hier ebenfalls so machen.

Reply:
In den Coding-Guidelines ist definiert, dass fachliche Benennungen in Deutsch und der Rest in Englisch geschrieben wird.